pysatMadrigal/instruments/gnss_tec.py

```diff
@@ -214,7 +214,19 @@ def load(fnames, tag=None, inst_id=None):
 
     # Fix the units for tec and dtec
     if tag == 'vtec':
-        meta['tec'] = {meta.labels.units: 'TECU'}
-        meta['dtec'] = {meta.labels.units: 'TECU'}
+        meta['tec'] = {meta.labels.units: 'TECU', meta.labels.min_val: 0.0,
+                       meta.labels.max_val: np.nan}
+        meta['dtec'] = {meta.labels.units: 'TECU', meta.labels.min_val: 0.0,
+                        meta.labels.max_val: np.nan}
+
+    # Get the maximum and minimum values for time, latitude, longitude,
+    # and altitude
+    meta['time'] = {meta.labels.notes: data['time'].values.dtype.__doc__,
+                    meta.labels.min_val: np.nan, meta.labels.max_val: np.nan}
+    meta['gdalt'] = {meta.labels.min_val: 0.0, meta.labels.max_val: np.nan}
+    meta['gdlat'] = {meta.labels.min_val: -90.0, meta.labels.max_val: 90.0}
+    min_lon = 0.0 if data['glon'].values.min() >= 0.0 else -180.0
```

Comment:
Do the longitudes differ across files?

Reply:
Not yet... but it seemed silly to not check for something that varies regularly from file iteration to file iteration and is very simple to verify.